indigo/bulk_creator.py

```diff
@@ -52,10 +52,13 @@ class RowValidationFormBase(forms.Form):
     commencement_date = forms.DateField(required=False, error_messages={'invalid': 'Date format should be yyyy-mm-dd.'})
```

Comment:
Maybe it's time we documented the spreadsheet columns on docs.laws.africa, link to that here, and also add some brief comments here. Without context, there are not easy to understand.

Reply:
done